geomstats/backend/numpy/__init__.py

```diff
@@ -203,3 +203,10 @@ def cumprod(x, axis=0):
 
 def copy(x):
     return x.copy()
+
+
+def from_vector_to_diagonal_matrix(x):
```

Comment:
Awesome! This vectorizes the numpy function `diag`, right?

I was just wondering if we should name it `diag`, but actually probably not: otherwise it might be confusing for users since it's not working the exact same way as the numpy function does.

Reply:
Exactly! But the name could be shortened anyway